test/scripts/jenkins_cloud.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# This script runs kibana tests compatible with cloud.
+#
+# The cloud instance setup is done in the elastic/elastic-stack-testing framework,
+# where the following environment variables are set pointing to the cloud instance.
+#
+# export TEST_KIBANA_HOSTNAME
+# export TEST_KIBANA_PROTOCOL=
+# export TEST_KIBANA_PORT=
+# export TEST_KIBANA_USER=
+# export TEST_KIBANA_PASS=
+#
+# export TEST_ES_HOSTNAME=
+# export TEST_ES_PROTOCOL=
+# export TEST_ES_PORT=
+# export TEST_ES_USER=
+# export TEST_ES_PASS=
+#
+
+set -e
+
+source "$(dirname $0)/../../src/dev/ci_setup/setup.sh"
+
+xvfb-run node scripts/functional_test_runner --debug --grep @skipcloud --invert $@
```

Comment:
I don't believe the tags exist in 5.6

Reply:
Correct there are no tests currently labelled as @skipcloud, but having the tag and the grep not finding anything is fine.  I had about 29 test failures running on 5.6.14, so after test analysis if we want to skip some tests we can just add the tag to the tests and not have to update the shell file -  so leaving it should not cause any issue.